test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
 			}
+			wg.Wait()
```

Comment:
the intent was to make the proxy deal with lots of different requests simultaneously... your change inverts that and makes the proxy deal with lots of identical requests simultaneously